test/unit/core/simd/view_to_simd_test.cpp

```diff
@@ -317,3 +331,31 @@ TYPED_TEST(view_to_simd_test, ends_not_on_chunk_boundary)
     auto simd_view = this->sequences | seqan3::views::to_simd<simd_t>;
     this->compare(simd_view, this->transformed_simd_vec);
 }
+
+// See https://github.com/seqan/seqan3/pull/1813
+// Before this fix we created a default constructed instance of the underlying range to use it as an empty range.
+// But a default constructed view is not guaranteed to be comparable, i.e. it results in UB.
+// This test checks now that if the underlying range is a view the conversion still works.
+TYPED_TEST(view_to_simd_test, default_constructed_empty_range_bug)
+{
+    using simd_t = typename TestFixture::simd_t;
+    using container_t = typename TestFixture::container_t;
+    using take_view_t = decltype(std::declval<container_t &>() | std::views::take(5));
+
+    std::vector<take_view_t> take_sequences{};
+
+    for (auto & sequence : this->sequences)
+        take_sequences.push_back(sequence | std::views::take(10));
+
+    // Remove one sequence
+    take_sequences.pop_back();
+
+    // delete the last sequence in the set.
+    for (simd_t & vec : this->transformed_simd_vec)
+        vec[seqan3::simd::simd_traits<simd_t>::length - 1] = TestFixture::padding_value_dna4;
+
+    { // without padding
```

Comment:
```suggestion
    { // without padding
```
? But isn't this with padding?

Reply:
Well, it is always with padding. But what I mean is with custom padding or not. So if not padded it will use some default value, which is the alphabet size I believe.

Reply:
What is custom padding? Ah I see this is terminology that you already introduced in the test. Okay!